Source/WebCore/css/parser/CSSPropertyParser.cpp

```diff
@@ -2771,6 +2771,45 @@ bool CSSPropertyParser::consumeWhiteSpaceShorthand(bool important)
     return true;
 }
 
+
+bool CSSPropertyParser::consumeAnimationRangeShorthand(bool important)
+{
+    // TODO: handle animation range sequences
+    RefPtr animationRangeStart = consumeAnimationRangeStart(m_range, m_context);
```

Comment:
This function is about the animation range so you don't need those words in the local variable names. This should just be named `start`.

Reply:
Fixed.